server/controllers/documents.js

```diff
@@ -0,0 +1,237 @@
+import { Users, Documents } from '../models';
+import {
+  catchError,
+  returnValidationErrors,
+} from '../helpers/utils';
+
+
+/**
+ * @description creates a document
+ * @function createDocument
+ * @param {object} req request object
+ * @param {object} res response object
+ * @returns {object} response object
+ */
+const createDocument = (req, res) => {
+  req.checkBody('title', 'fullname cannot be empty').notEmpty();
```

Comment:
This is handled by the express-validator package. It returns an array of objects containing errors